### ISSUE TITLE
Symbol 'type scala.annotation.nowarn' is missing from the classpath

### ISSUE THREAD

**Issue by patriknw** [akka/akka#30018]
When compiling an application project with latest Akka (2.6.12+11-b41d6a0c) I see the following:

```
Symbol 'type scala.annotation.nowarn' is missing from the classpath.
[error] This symbol is required by ' <none>'.
[error] Make sure that type nowarn is in your classpath and check for conflicting dependencies with `-Ylog-classpath`.
[error] A full rebuild may help if 'Eventsourced.class' was compiled against an incompatible version of scala.annotation.
```

same after clean

Akka compiled with Scala 2.12.13.
Application compiled with Scala 2.12.12

It disappears when changing scalaVersion in the application to 2.12.13.

I guess this is because of the Scala 2.12.12 -> 2.12.13 update in Akka.

Probably nothing we can/should do aside from mentioning in release notes of Akka 2.6.13.

cc @lukas-ryt

**Comment by patriknw** [comment]
Closing without action, but we should mention it in release notes.

**Comment by patriknw** [comment]
Perhaps this is a problem with Scala 2.13.4 also? I don't think I tried the same project with that. Noticed that something related was fixed in Scala 2.13.5 https://github.com/scala/scala/pull/9491
